.circleci/ci-helper.sh

```diff
@@ -1,9 +1,25 @@
 #!/usr/bin/env bash
 # inspired by https://github.com/Originate/guide/blob/master/android/guide/Continuous%20Integration.md
 
+function envSetup {
+    sudo apt-get update
+    sudo apt-get install libqt5widgets5
+    sudo npm install -g shelljs@0.7.0
+    sudo npm install -g cordova
```

Comment:
Should we install a specific version (e.g 7.0.0)?

Reply:
+1. `cordova@7.0.0` would be ideal.